app/controllers/hyrax/valkyrie_file_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Hyrax
+  class ValkyrieFileController < ApplicationController
```

Comment:
i think this still needs authentication, and i'm wondering what the difference between this and `DownloadsController` is.

Reply:
i'm happy to combine them if we can